packages/elastic_package_registry/_dev/build/build.yml

```
@@ -0,0 +1,3 @@
+dependencies:
+  ecs:
+    reference: git@v8.0.0
```

In case there is any confusion, we are proposing to use the latest version here https://github.com/elastic/integrations/pull/3671#discussion_r928231675
```suggestion
    reference: git@v8.3.1
```

---

Updated to `git@v8.3.1`